packages/namekit-react/tsup.config.js

```diff
@@ -3,7 +3,10 @@ import { defineConfig } from "tsup";
 export default defineConfig({
   entry: ["src/index.ts"],
   format: ["esm"],
+  external: ["react", "react-dom"],
```

Comment:
Interesting. I assumed we wouldn't need to specify anything explicit as `external`? As I understand any dependencies in package.json are automatically excluded? https://tsup.egoist.dev/#excluding-packages